index.html

```diff
@@ -762,9 +762,311 @@
         <!-- ************ -->
 
         <section id="class-3">
-          <h2>Third Class: Jeff</h2>
+          <h2>Third Class</h2>
+          <p>Jeff O'Leary</p>
         </section>
+        <section>
+          <h4>Agenda</h4>
+          <p class="fragment">Navigation</p>
```

Comment:
What are some core concepts you would suggest reviewing?  I can add more
notes that review them or just list them on a slide.

On Sunday, June 14, 2015, Elliot Schrock <notifications@github.com> wrote:

> In index.html
> <https://github.com/gdiboston/gdi-intro-ios/pull/8#discussion_r32387148>:
>
> >          </section>
> > +        <section>
> > +          <h4>Agenda</h4>
> > +          <p class="fragment">Navigation</p>
>
> That's fine. Just want to reinforce core concepts since they seem fuzzy on
> it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdiboston/gdi-intro-ios/pull/8/files#r32387148>.
>


-- 
Best Regards,
Jeff O'Leary


[image: MeYou Health][image: FB] <https://www.facebook.com/meyouhealth>[image:
T] <http://twitter.com/#!/meyouhealth>[image: RSS]
<http://www.meyouhealth.com/blog/>[image: YT]
<http://www.youtube.com/user/meyouhealthvideos>[image: F]
<http://www.flickr.com/photos/meyouhealth>
*Jeff O'Leary*iOS Engineer
Cell: 857-268-2939
MeYou Health - the social well-being companywww.meyouhealth.com
jeff.oleary@meyouhealth.com
MeYou Health is a Healthways, Inc. company